spec/models/journaled/client_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Journaled::Client do
```

Comment:
Most of this was extracted from `delivery_spec.rb` as was the corresponding logic